nn_sdwan/tox.ini

```
@@ -0,0 +1,25 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py39
+envlist =
+    py{27,38,39}
+
+[testenv]
+ensure_default_envdir = true
+envdir =
+    py27: {toxworkdir}/py27
+    py38: {toxworkdir}/py38
+    py39: {toxworkdir}/py39
```

```suggestion
    py38: {toxworkdir}/py38
```
It seems you're using py3 syntax so the tests fail to run with python2.
Also the agent does not still support py3.9